src/components/Link/Link.jsx

```diff
@@ -21,13 +21,14 @@ Link.propTypes = {
   addClass: PropTypes.string,
   href: PropTypes.string.isRequired,
   icon: PropTypes.string,
-  label: PropTypes.string.isRequired,
+  label: PropTypes.string,
```

Comment:
we shouldn't allow both the icon and label to be blank in this component. although it is introduced in this pr, it will touch a few files unrelated to the scope of this pr. so, I will create a new ticket to address that on.

Reply:
ref: https://github.com/scientist-softserv/webstore-component-library/issues/165